src/app/routes/profile/content/editprofile.js

```diff
@@ -0,0 +1,252 @@
+                                                            <select name="phonetype" ref="phonetype">
+                                                                <option value="" disabled selected>Phone Type</option>
+                                                                {phone_types.map(function(type) {
+                                                                        return(
+                                                                            <option key={type} value={type}>{type}</option>
+                                                                        )
+                                                                    })
+                                                                }
+                                                            </select><i/>
+                                                        </label>
+                                                    </section>
+                                                </div>
+                                            </fieldset>
+                                            <fieldset>
+                                                <br/>
+                                                <div className="row">
+                                                    <section className="col col-6">
+                                                            <label htmlFor="address2" className="input">
+                                                                <input type="text" name="street1" ref="street1" placeholder="Street 1"/>
+                                                            </label>
+                                                    </section>
+                                                    <section className="col col-6">
+                                                        <label htmlFor="address2" className="input">
+                                                            <input type="text" name="street2" ref="street2" placeholder="Street 2"/>
+                                                        </label>
+                                                    </section>
+                                                </div>
+                                                <div className="row">
+                                                    <section className="col col-6">
+                                                        <label className="input">
+                                                            <input type="text" name="city" ref="city" placeholder="City"/>
+                                                        </label>
+                                                    </section>
+                                                    <section className="col col-2">
+                                                        <label className="input">
+                                                            <input type="text" name="state" ref="state" placeholder="State"/>
```

Comment:
set a maximum string length for state or consider adding in a drop down select option